### PR TITLE
interfaces/builtin/fwupd.go: allow access to nvmem for thunderbolt plugin

### PR DESCRIPTION
**Pull request by valentindavid**
Downstream issue: https://github.com/fwupd/fwupd/issues/8014

### CODE REVIEW

File: interfaces/builtin/fwupd.go [hunk 1]
```diff
@@ -154,6 +154,10 @@ const fwupdPermanentSlotAppArmor = `
   /sys/devices/**/psp_vbflash rw,
   /sys/devices/**/psp_vbflash_status r,
 
+  # Required by plugin thunderbolt
+  /sys/devices/**/nvm_non_active*/nvmem a,
```
Comment (bboozzoo) [inline interfaces/builtin/fwupd.go:158]: do we know if this should maybe allow write instead of only append?

**Review by bboozzoo — APPROVED** [review]
Thanks

File: interfaces/builtin/fwupd.go (hunk 1 above)
Comment (valentindavid) [reply interfaces/builtin/fwupd.go:158]: I think the only place we use the "non_active" one is here:

https://github.com/fwupd/fwupd/blob/65e5fcf3bef12a475e8b9383bab35775b9139fd2/plugins/thunderbolt/fu-thunderbolt-device.c#L307

Look at the calls to `fu_thunderbolt_device_find_nvmem`. The second parameter is a boolean. If it is "FALSE"  it is a "non_active" one.

Comment (valentindavid) [reply interfaces/builtin/fwupd.go:158]: Ah, now I understand, "a" is only O_APPEND, where was "w" is both. But you cannot have "aw".

Comment (valentindavid) [reply interfaces/builtin/fwupd.go:158]: I still think it would be safer to only allow O_APPEND.

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM

**Review by alexmurray — APPROVED** [review]
LGTM!

**Pull request merged**